crates/astria-bridge-withdrawer/src/withdrawer/ethereum/watcher.rs

```diff
@@ -250,6 +260,32 @@ impl Batcher {
                     info!("batcher shutting down");
                     break;
                 }
+                block = block_rx.next() => {
```

Comment:
with adding this, should we remove the functionality which sends on receiving an event that occurs in a new block. It seems very duplicative? New events push into the batches, new blocks send them?

Reply:
i think we need to leave the event logic which pushes on new block, for example, if we get an event that's for a newer block than the current batch before we get the notification of the new block. then, the batch will end up containing events from multiple block numbers, which is not what we want